src/styles/c-header.scss

```diff
@@ -182,28 +241,22 @@ $link-color: #747472;
     border-left: 0;
     border-bottom: 0;
     background-color: transparent;
-    transition: top 0.2s ease-out;
 
     &:before {
       width: 48px;
       height: 46px;
     }
+    
   }
-  :host([stuck]) {
-
-    .navbar-symbol{
-      margin-top: 18px;
-      top:65px;
-      &:before{
-        width: 40px;
-        height: 38px;
-      }
-    }
+}
 
-    ::slotted(c-navigation),
-    &::slotted(c-navigation) {
-      -webkit-box-shadow: 0px 17px 15px -15px rgba(0, 0, 0, 0.15);
-      box-shadow: 0px 17px 15px -15px rgba(0, 0, 0, 0.15);
+// IE special handling
+@media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {

```

Comment:
Try to set the position static as a general rule for all slotted a nodes in all browsers

Reply:
Ok, I change some properties so we don't need this rule anymore